Packs/CommonScripts/Scripts/IsolateEndpoint/IsolateEndpoint.yml

```diff
@@ -0,0 +1,62 @@
+args:
+- description: List of agent IDs of the endpoint to isolate.
+  name: agent_id
+  isArray: true
+- description: List of agent IPs of the endpoint to isolate.
+  name: agent_ip
+  isArray: true
+- description: List of agent hostnames of the endpoint to isolate.
+  name: agent_hostname
+  isArray: true
+- description: Should the isolate be force.
+  name: force
+- description: |-
+    Specify the integration brands to run the command for. If not provided, the command will run for all available integrations. For multi-select, provide a comma-separated list. For example: 'Active Directory Query v2, CrowdstrikeFalcon, ExtraHop v2'.
```

Comment:
The integrations in the example do not match the integrations in the predefined list.  Is that a problem?